bloom-instance/main.tf

```diff
@@ -66,11 +56,35 @@ module "public_alb" {
       use_tls     = false
       allowed_ips = ["0.0.0.0/0"]
     }
+
+    # internal is here just to provide an easy path forward if we want an internal route to services
     internal = {
       port        = 8080
       use_tls     = false
       allowed_ips = [for subnet in module.network.subnets.app : subnet.cidr_block]
     }
   }
+}
+
+module "public_sites" {
+  for_each = { for idx, srv in var.public_sites : idx => srv }
+  source   = "./services/public-site"
 
+  name_prefix        = local.default_name
+  service_definition = each.value
+
+  alb_listener_arn = module.public_alb.listeners.public.arn
+  alb_sg_id        = module.public_alb.security_group.id
+  subnet_ids       = [for subnet in module.network.subnets.app : subnet.id]
+
+  public_upload_bucket = aws_s3_bucket.user_upload_bucket.bucket
+  secure_upload_bucket = aws_s3_bucket.user_upload_bucket.bucket
```

Comment:
Yes.  At the moment there is only one bucket defined, so that's the one I used.  I'm going to submit a different PR creating distinct buckets with distinct access policies, but for now I wanted to make sure that would have minimal impact on the service definitions.